client.go

```diff
@@ -245,7 +245,8 @@ func NewVersionedTLSClientFromBytes(endpoint string, certPEMBlock, keyPEMBlock,
 		tlsConfig.RootCAs = caPool
 	}
 	tr := &http.Transport{
-		TLSClientConfig: tlsConfig,
+		TLSClientConfig:   tlsConfig,
+		DisableKeepAlives: true,
```

Comment:
careful, this will make reuse of a single transport much more expensive (re-establishing a TLS connection for every request)